instrumentation/opentelemetry-instrumentation-pymysql/src/opentelemetry/instrumentation/pymysql/__init__.py

```diff
@@ -50,10 +50,10 @@
 from opentelemetry.instrumentation.pymysql.version import __version__
 
 _CONNECTION_ATTRIBUTES = {
-    "database": "db",
-    "port": "port",
-    "host": "host",
-    "user": "user",
+    "db.namespace": "db",
```

Comment:
These are not the correct attributes to be changing for semantic conventions. Pymysql (as well as several other db instrumentations) use dbapi under the hood for it's instrumentations. You must check whether [dbapi](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py#L315) span attributes conform to the most recent db semantic conventions. dbapi is also one of the instrumentations needed for the [semantic convention migration plan](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2351#issuecomment-2420283949), so it is not as simple as changing the attributes to new values.

Reply:
Should I ensure that dbapi adheres to [these](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md) semantic conventions?

Reply:
@Joice-crypto 

This issue is a bit difficult for new contributors to pick up as it involves implementing the [semantic convention migration plan](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2351#issuecomment-2420283949). I would advise picking up a different issue if you are trying to pick up good first issues,